### DISCUSSION TITLE
Error in autentication - cannot access project

### DISCUSSION THREAD

**Discussion by asiergonio** [AntlerVC/firetable#1027] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**

Error code: 500: //developers.google.com/identity/sign-in/web/devconsole-project. Raw server response: "{"error":{"code":401,"message":"Request had invalid authentication credentials. Expected OAuth 2 access token, login cookie or other valid authentication credential. See https://developers.google.com/identity/sign-in/web/devconsole-project.","errors":[{"message":"Invalid Credentials","domain":"global","reason":"authError","location":"Authorization","locationType":"header"}],"status":"UNAUTHENTICATED","details":[{"@type":"type.googleapis.com/google.rpc.ErrorInfo","reason":"ACCOUNT_STATE_INVALID","domain":"googleapis.com","metadata":{"email":"rowy-service@******.gserviceaccount.com","service":"identitytoolkit.googleapis.com","method":"google.cloud.identitytoolkit.v1.AccountManagementService.GetAccountInfo"}}]}}"

**Comment by BeeBombshell** [comment, marked as answer]
Hey @asiergonio, please make sure that you have the required Firestore Rules set. For more information refer: https://docs.rowy.io/how-to/roles#write-firestore-security-rules-with-user-roles

Also, make sure you have your account set with the `ADMIN` role using the instructions on that page.
